doc/nrf/gs_dtm_testing.rst

```diff
@@ -0,0 +1,118 @@
+.. _gs_dtm_testing:
+
+Direct Test Mode terminal connection
+====================================
+
+.. contents::
+   :local:
+   :depth: 2
+
+Follow the instructions in this section to connect with the DK which is running the DTM to send commands and receive responses.
+
+The Bluetooth Low Energy DTM UART interface standard specifies the following configuration:
+
+* Eight data bits
+* No parity
+* One stop bit
+* No hardware flow control
+* A selection of bit rates from 9600 to 1000000, one of which must be supported by the DUT.
+  It might be possible to run other bit rates by experimenting with parameters.
+
+.. note::
+   The default bit rate of the DTM UART driver is 19200 bps, which is supported by most certified testers.
+
+You have to send all commands as a two-byte Hex number. The responses must have the same format.
+
+Connect with RealTerm (Windows)
+*******************************
+
+The RealTerm start window:
+
+.. figure:: /images/realterm.png
+   :alt: RealTerm start window
+
+To test the DTM with the RealTerm, complete the following steps:
+1. On the :guilabel:`Display` tab, set :guilabel:`Display As` to :guilabel:`Hex[space]`.
+
+   .. figure:: /images/realterm_hex_display.png
+      :alt: Set the RealTerm display format
+
+2. Open the :guilabel:`Port` tab and configure the serial port parameters:
+
+   * Set the :guilabel:`Baud` to 19200 **(1)**
+   * Choose your Jlink serial port from the :guilabel:`Port` list **(2)**
+   * Click twice the :guilabel:`Open` button **(3)**
+
+   .. figure:: /images/real_term_serial_port.png
+      :alt: RealTerm serial port settings
+
+3. Open the :guilabel:`Send` tab:
+
+   a. Write the command as Hex number in the field **(1)**.
+      For example, write **0x00 0x00** to send a **Reset** command.
+   b. Click the :guilabel:`Send Numbers` button **(2)** to send the command.
+   c. Observe the response in the DTM in area **(3)**.
+      The response is encoded as Hex numbers.
+
+   .. figure:: /images/realterm_commands.png
+      :alt: RealTerm commands sending
+
+Connect with minicom (Linux)
+****************************
+
+On the Linux operating system, install a Minicom terminal.
+On Ubuntu, run:
+
+.. code-block::
+
+   sudo apt-get install minicom
+
+1. Run the Minicom terminal:
+
+   .. code-block::
+
+      sudo minicom -D <DTM serial port> -s
+
+   For example:
+
+   .. code-block::
+
+      sudo minicom -D /dev/serial/by-id/usb-SEGGER_J-Link_000683580193-if00 -s
+
+   The **-s** option switches you to Minicom setup mode.
+
+2. Configure the Minicom terminal:
+
+   Configuration window:
```

Comment:
You might want to have this text as the image caption instead of having it before the image.